scripts/update-3rd-party.ts

```diff
@@ -72,6 +72,23 @@ const prepManifestSchema = async (content: string, location: string): Promise<st
     return JSON.stringify(schema, null, 4);
 };
 
+/**
+ * ajv 7 doesn't support property id. It uses $id.
+ */
+const replaceId = (content: string, location: string): string => {
+    const schema = JSON.parse(content);
+
+    if (!('$id' in schema)) {
+        schema.$id = schema.id;
+    }
+
+    if ('id' in schema) {
+        delete schema.id;
+    }
+
+    return JSON.stringify(schema, null, 4);
+};
```

Comment:
Can/should we also open a PR to the schema repo to apply these changes there? I'm okay with this as a quick-fix, but I'm hoping we can get rid of it as soon as the source schemas are up-to-date.

Reply:
Oh yeah, I forgot the schema repo is still sitting at schema version 4 and not 7 so they may not be willing to update quickly. Given that I wouldn't recommend doing more than filing a bug with them and we'll carry on here.